src/providers/annotations.ts

```diff
@@ -101,7 +101,7 @@ function clearEvaluationDecorations(editor?: vscode.TextEditor) {
             editor.document.uri + ':resultDecorationRanges'
         );
         setResultDecorations(editor, []);
-        for (const status in [
+        for (const status of [
```

Comment:
This accidentally worked since the values for this AnnotationStatus enum are `[0, 1, 2, 3]` and the indexes that would have been the value of status with for/in would have been `["0", "1", "2", "3"]`